bin/plotting/pycbc_page_snrratehist

```diff
@@ -145,17 +146,27 @@ histpeak = bins[exc_binvals[0].argmax()]
 # plot full background
 if not args.closed_box:
     if h_inc_back_num == 0:
-        pylab.hist(bstat, bins=bins, histtype='step',
-                   linewidth=2,
-                   color='black', log=True,
-                   label='Full Background',
-                   weights=dec / f.attrs['background_time'] * lal.YRJUL_SI)
+        pylab.hist(
+            bstat,
+            bins=bins,
+            histtype='step',
+            linewidth=2,
+            color='black',
+            log=True,
+            label='Full Background',
+            weights=dec / conv.sec_to_year(f.attrs['background_time'])
```

Comment:
this looks correct, but it is a bit messy, as in fact it was before .. ideally one would define eg `w = dec / conv.sec_to_year(f.attrs['background_time'])` and then just `weights=w` in the hist call.